### ISSUE TITLE
Auto upload

### ISSUE THREAD

**Issue by MeikelLP** [qcasey/paperless_share#9]
I use the Brother iPrint&Scan app which is putting my scans into Downloads/Scan.

I don't know whether I can configure that but it would be awesome if this app could automatically get files from that directory and push it to paperless.

I don't know if this is possible tho as I am not an android dev

**Comment by qcasey** [comment]
Interesting idea, but it seems hard to accomplish right in practice.

To do this, the app would need to keep in sync with the server. Right now it simply fires off the document and hopes for the best, regardless of OCR issues, filesystem issues, etc. It'd need to know what the server has already received and processed.

Although I will keep this in mind for future updates.

In the meantime I'd recommend setting up [Syncthing](https://play.google.com/store/apps/details?id=com.nutomic.syncthingandroid&hl=en_US&gl=US) to sync Downloads/Scan from your device to the `consume` directory on the paperless server.

**Comment by MeikelLP** [comment]
My idea is more like a "consume directory" than a sync directory.

Like u can can make any directory on the paperless server a consume directory. I'd love to see a similar consume  directory on Android

**Comment by qcasey** [comment]
Right, but even then we'd still have to know what the server has processed in order to safely delete files from the local consume directory. 

I'm open to it, but I don't have that much time to dedicate to this app. 

I bring up Syncthing because it already does this. Once it syncs to the server's consume directory, paperless will process and safely delete the file. Syncthing will notice that change and delete it from your device as well. Effectively consuming it

**Comment by MeikelLP** [comment]
From what I have seen in paperless already processed files (i.e. duplicates) are ignored in the consume directory once an error occurs. For me this would be ok as well. Maybe a list in the app that show "error files"?

Just ideas. Please don't be stressed :)
